cds-snc/cppd-medical-report-sails
# Medical practitioner Your information section

# What this does

Creates a new Practitioner Information section.
- Move Practitioner information from declaration to its own section
- Dashboard validation for Practitioner information
- Added new fields from the design
- Left placeholders in for the Declaration

## To test
- Start a medical report
- Login to the doctor/medical practitioner section
- Notice the new section on the dashboard
- Test out the new section

## NOTE
This section replaces the existing Declaration page - I haven't built out the new Declaration section, I've just left placeholders in those files for a future PR.
